code/_onclick/click.dm

```diff
@@ -32,9 +32,13 @@
 */
 /atom/Click(location,control,params)
 	usr.ClickOn(src, params)
+
 /atom/DblClick(location,control,params)
 	usr.DblClickOn(src,params)
 
+/atom/MouseWheel(delta_x,delta_y,location,control,params)
+	usr.MouseWheelOn(src, delta_y)

```

Comment:
I think I'll keep with no modifier for now. But I'll pass the other args in case someone wants to use it for something else.